### PR TITLE
adding-itemcounter-price

### PR DESCRIPTION
**Pull request by garcid1663**

### CODE REVIEW

File: finalProject.html [hunk 1]
```diff
@@ -7,6 +7,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="finalProject.css">
     <title>Store</title>
+    <script src="store.js" async></script>
```
Comment (johngarcia9110) [inline finalProject.html]: this should be placed just before the </body> tag below. This helps make sure that you JS code can reference the code above when its loaded.

**Review by johngarcia9110 — APPROVED** [review]
LGTM! One little nit but overall seems good! Nice job dude

File: finalProject.html (hunk 1 above)
Comment (garcid1663) [reply finalProject.html]: Should be placed above the closing head tag? Or placed above title?

Comment (johngarcia9110) [reply finalProject.html]: Weird I swear I typed an example of the tag lol sorry, right before/above the closing body tag

File: finalProject.html [hunk 2]
```diff
@@ -34,8 +35,36 @@ <h2>Shopping-Cart</h2>
...
+
 
         </div>
+        <div class="item4">
+            <h2>Cart</h2>
+            <p>Items: <span id="itemCount">0</span></p>
+            <p>Total price: $<span id="totalPrice">0</span></p>
+        </div>
     </div>
 </body>
```
Comment (johngarcia9110) [inline finalProject.html:70]: Right above here

**Review by johngarcia9110 — APPROVED** [review]

File: finalProject.html (hunk 1 above)
Comment (garcid1663) [reply finalProject.html]: ah okay! Sounds good brother man thank ya!

**Pull request merged**